content/source/docs/enterprise/api/configuration-versions.html.md

```diff
@@ -60,19 +72,119 @@ curl \
 
 -> **Note**: Uploading a configuration file automatically creates a run and associates it with this configuration-version. Therefore it is unnecessary to [create a run on the workspace](./run.html#create-a-run) if a new file is uploaded.
 
-| Method | Path           |
-| :----- | :------------- |
-| POST | The upload URL is provided in the `upload-url` attribute in the `configuration-versions` resource |
+`POST https://archivist.terraform.io/v1/object/4c44d964-eba7-4dd5-ad29-1ece7b99e8da`
```

Comment:
This would be a PUT for Archivist.

Reply:
Also, we should decorate this to emphasize that it's not copy-pasteable from the docs! I suggest replacing the real object ID here with a placeholder. Like this: 

`PUT https://archivist.terraform.io/v1/object/<UNIQUE OBJECT ID>`

The different hostname for the archivist service might also be confusing for readers on PTFE, but I'm not sure? It's probably fine.